### DISCUSSION TITLE
error fatal: repository <URL> not found

### DISCUSSION THREAD

**Discussion by lientjelee** [community/community#41495] [New to GitHub] [answered]
Hi everyone, 

I'm totally new to github and trying to figure this error out. 
I'm trying to push my first code to my own private repo from the terminal (Mac), but experiencing the error "repository not found".
I've got git installed (v 2.37.1 (Apple Git-137.1) and gh version 2.20.2 (2022-11-15). 
after the "git push -u origin main" command it will ask for my credetials, 
I log in with my PAT and then I get the error "fatal: repository not found". 
The PAT has acces to all the the scopes. 

I did a test with a public repo and that works just fine.

**Comment by lientjelee** [comment]
addition: I already deleted all the github data that was saved in my keychain

**Comment by lientjelee** [comment, marked as answer]
nm, I fixed it by deleting the origin and adding it again. Although I copied the URL from github, the problem was the URL

**Reply by queenofcorgis** [reply]
Nice self solve!

**Reply by pipelee24** [reply]
Saya tidak DPT mengload game

**Comment by pipelee24** [comment]
Saya tidak dapat mengload App game
